working/unquoted-imports/feature-specification.md

```diff
@@ -291,27 +355,33 @@ We add a new rule and hang it off the existing `uri` rule already used by import
 and export directives:
 
 ```
-uri                   ::= stringLiteral | packagePath
-packagePath           ::= packagePathSegment ( '/' packagePathSegment )*
-packagePathSegment    ::= dottedIdentifierList
-dottedIdentifierList  ::= identifier ('.' identifier)*
+uri               ::= stringLiteral | packagePath
+packagePath       ::= pathSegment ( '/' pathSegment )*
+pathSegment       ::= segmentComponent ( '.' segmentComponent )*
+segmentComponent  ::= identifier
+                    | ⟨RESERVED_WORD⟩
+                    | ⟨BUILT_IN_IDENTIFIER⟩
+                    | ⟨OTHER_IDENTIFIER⟩
 ```
 
-An import or export can continue to use a `stringLiteral` for the quoted form
-(which is what they will do for relative imports). But they can also use a
-`packagePath`, which is a slash-separated series of segments, each of which is a
-series of dot-separated identifiers. *(The `dottedIdentifierList` rule is
-already in the grammar and is shown here for clarity.)*
+It is a compile-time error if any whitespace, newlines, or comments occur
+between any of the `segmentComponent`, `/`, or `.` tokens in a `packagePath`.
```

Comment:
If I was implementing the parser, I would simply parse the grammar ignoring comments or whitespace between the tokens like you do in most places. Then I'd go back and report errors if there were comments or whitespace between any of the terminals.

So the proposal is pretty much specified the same way.

I don't think it matters too much one way or the other. It might matter a lot of we make semicolons optional, but I think we can cross that bridge when we come to it. I'll leave it as is and if the parser folks ask questions, I can tweak it.